src/deploy/standalone/noobaa_syslog.conf

```diff
@@ -4,6 +4,17 @@ $umask 0000
 $FileCreateMode 0644
 $EscapeControlCharactersOnReceive off
 
+# Remove defaul ratelimiting, Setting this value to 0 turns off ratelimiting
+$imjournalRatelimitInterval 0
+$imjournalRatelimitBurst 0
+
+# start log rotation via outchannel
+# outchannel definition, 100M limit, If updating this value update logrotate also.
+$outchannel log_rotation,/var/log/noobaa.log,104857600,/usr/local/noobaa-core/src/deploy/standalone/logrotate_noobaa
+#  activate the channel and log everything to it
+local0.* :omfile:$log_rotation
```

Comment:
If I understand correctly, having both this line and the line 43 would produce the messages twice in the file - can you check if that's the case? I expected the outchannel to replace line 43.

Reply:
You are right same logs are generated twice in log file. Will remove the line 43

Reply:
but this means you also need a separate channel for the noobaa_events too for the local2 facility

Reply:
No, Its happening for noobaa logs because we mention it in two places and its not the case for noobaa event logs

Reply:
but how would noobaa_events get rotated by size then?